We do not need to run performance tests for this project.

--- importlib_metadata-9.0.1/pyproject.toml.orig
+++ importlib_metadata-9.0.1/pyproject.toml
@@ -38,7 +38,6 @@
 	# local
 	"packaging",
 	"pyfakefs",
-	"pytest-perf >= 0.17",
 ]
 
 doc = [
